core-bundle/src/Routing/Page/PageRouteProviderInterface.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Routing\Page;
+
+use Contao\PageModel;
+use Symfony\Component\Routing\Route;
+
+/**
+ * A page route provider provides routing information for a page type.
+ * The route provider knows what route for a page of its type looks like, e.g. if
+ * a particular route has a different URL suffix than configured in the root page.
+ */
+interface PageRouteProviderInterface
+{
+    /**
+     * While matching URLs, contao generates alias candidates and looks for matching page models.
```

Comment:
```suggestion
     * While matching URLs, Contao generates alias candidates and looks for matching page models.
```

Reply:
Fixed in de480f2f5314037da6f0ecb203fc77d093e6a205